pkg/image/bootc_disk.go

```diff
@@ -5,26 +5,34 @@ import (
 	"math/rand"
 
 	"github.com/osbuild/images/pkg/container"
+	"github.com/osbuild/images/pkg/customizations/users"
+	"github.com/osbuild/images/pkg/disk"
 	"github.com/osbuild/images/pkg/manifest"
 	"github.com/osbuild/images/pkg/osbuild"
+	"github.com/osbuild/images/pkg/platform"
 	"github.com/osbuild/images/pkg/runner"
 )
 
 type BootcDiskImage struct {
-	*OSTreeDiskImage
+	Base
+
+	Platform       platform.Platform
+	PartitionTable *disk.PartitionTable
+
+	// This is a bit of a lie, only root and it's ssh key is supported
+	// today because that is all that bootc gives us by default but
```

Comment:
Updated it to say
```
	// "Users" is a bit misleading as only root and its ssh key is supported
	// right now because that is all that bootc gives us by default but that
	// will most likely change over time.
	// See https://github.com/containers/bootc/pull/267
```